pkgs/by-name/gi/git-pr/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "git-pr";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "picosh";
+    repo = "git-pr";
+    rev = "v${version}";
+    hash = "sha256-7Ka8p5X8nQBXKiT6QsWOWMQJL8rePKrHz/LZU1W+oQ8=";
+  };
+
+  vendorHash = "sha256-tu5C7hz6UTgn/jCCotXzZHlUmGVNERhA7Osxi31Domk=";
+
+  postInstall = ''
+    mv $out/bin/ssh $out/bin/git-pr
```

Comment:
Maybe git-ssh is better, as suggested in https://github.com/picosh/git-pr/tree/v0.0.2?tab=readme-ov-file#docker. Also there's a `web` which is the server side. Is that also needed? (The server side also needs some resource files)

Reply:
Agreed. @Sigmanificient ping